cypress/utils/utils.ts

```diff
@@ -1652,7 +1653,7 @@ export function enumKeys<O extends object, K extends keyof O = keyof O>(obj: O):
 }
 
 export function getUrl(): string {
-    return window.location.href;
+    return Cypress.env("tackleUrl");
```

Comment:
@ibragins @sshveta could you please confirm this change?

Reply:
Both should work @mguetta1 . 
Windows location href is getting url from the opened window which should be same as cypress.env(tackleUrl).

I don't see a need for this change .